content/glance.md

```diff
@@ -56,11 +65,11 @@ Guidelines (ATAG) 2.0. For the normative technical specification, see
 
 -   Make alternative content available to authors
 -   Ensure that editing-view presentation can be [[programmatically
```

Comment:
@shawna-slh Any idea why "programmatically determined" is under square brackets? Is it voluntary, or can it be removed?

Reply:
Fixed!